_data/sites.json

```
@@ -811,6 +821,15 @@
         ]
     },
 
+    {
+        "name": "Boxcryptor",
+        "url": "https://www.boxcryptor.com/app/account/delete-account/",
+        "difficulty": "easy",
+        "domains": [
+            "www.boxcryptor.com"
```

```suggestion
            "boxcryptor.com"
```
No need for the "www"